titus-ext/zookeeper/src/test/java/com/netflix/titus/ext/zookeeper/supervisor/ZookeeperMasterMonitorTest.java

```diff
@@ -146,7 +148,12 @@ public void testLocalMasterInstanceUpdates() throws Exception {
     private void expectMasters(ExtTestSubscriber<List<MasterInstance>> mastersSubscriber, MasterInstance... masters) throws Exception {
         List<MasterInstance> last = null;
         for (List<MasterInstance> next; (next = mastersSubscriber.takeNext()) != null; ) {
```

Comment:
do we need a `.timeout()` here on the `takeNext()`?

Reply:
I had to rewrite this code. Please, check my next PR.
There is different ZK behavior in Travis in when running on my Mac.